src/bun.js/api/server.zig

```diff
@@ -4834,6 +4834,13 @@ pub fn NewServer(comptime NamespaceType: type, comptime ssl_enabled_: bool, comp
         temporary_url_buffer: std.ArrayListUnmanaged(u8) = .{},
 
         cached_hostname: bun.String = bun.String.empty,
+        cached_localAddr: struct {
```

Comment:
Using a getter, we won't need to cache it here

Instead, the `JSValue` will only be created once

Reply:
Fixed. Now using a getter and `cache: true`.